add auto expand groups property (#1172)
telerik/maui-docs: controls/datagrid/grouping/expand-collapse.md

# Expand and Collapse Groups in .NET MAUI DataGrid

> The [Telerik UI for .NET MAUI DataGrid]({%slug datagrid-overview%}) supports group expand and collapse operations either through the UI by tapping on the group headers, or programmatically. 

Reviewer: ```suggestion
The [Telerik UI for .NET MAUI DataGrid]({%slug datagrid-overview%}) lets you expand and collapse a group either through the UI&mdash;by tapping on the group headers&mdash;or programmatically. 
```
